template/solutions/clis.html

```diff
@@ -0,0 +1,226 @@
+                </div>
+                <p class="MediaList-itemBoxBody-top">MongoDB <a href="https://www.mongodb.com/blog/post/go-agent-go" rel="noreferrer" target="_blank">使用 Go 实现了他们的后端 CLI 工具</a>，因为 Go 的众多优点：类 C 语法，强大的标准库，可以通过 goroutines 解决的并发场景，和无痛的多平台发布。</p>
+            </li>
+
+            <li class="MediaList-itemBox">
+                <div class="MediaList-itemBoxHeader-top">
+                    <a href="https://kubernetes.com" target="_blank"><img class="MediaList-itemBoxImage" src="/static/imgs/logos/kubernetes.svg" alt="Kubernetes Logo"></a>
+                    <p class="MediaList-itemBoxTitle"></p>
+                </div>
+                <p class="MediaList-itemBoxBody-top">Kubernetes 是最受欢迎的用 Go 写的 CLI 应用之一。 <a href="https://blog.gopheracademy.com/birthday-bash-2014/kubernetes-go-crazy-delicious/" rel="noreferrer" target="_blank">Kubernetes 的创造者，Joe Beda, 说对于 Kubernetes</a>, &ldquo;Go 是唯一的最佳选择&rdquo;。可以说在低级语言（像 C++）和高级语言（像 Python）之间选择，Go 是最佳的。</p>
+            </li>
+
+            <li class="MediaList-itemBox">
+                <div class="MediaList-itemBoxHeader-top">
+                    <a href="https://gohugo.io" target="_blank"><img class="MediaList-itemBoxImage" src="/static/imgs/logos/hugo.svg" alt="Hugo Logo"></a>
+                    <p class="MediaList-itemBoxTitle"></p>
+                </div>
+                <p class="MediaList-itemBoxBody-top">Hugo 是支持数千站点的最流行的 Go CLI 应用程序之一。 它受欢迎的一个原因就是 Go 的易于安装。 Hugo 作者 <a href="https://gohugo.io/news/lets-celebrate-hugos-5th-birthday/" rel="noreferrer" target="_blank">Bjørn Erik Pedersen 写到</a> &ldquo;单一的二进制文件解决了安装和升级的大部分问题。&rdquo;</p>
+            </li>
+
+
+
+        </ul>
+
+        <h2 id="how-to-use-go-for-clis">如何使用 Go 开发 CLI 应用程序</h2>
+
+        <p>在使用 Go 开发 CLI 时，有两种工具被广泛使用：Cobra &amp; Viper 。</p>
+
+        <div class="PullQuote">
+            <div class="PullQuote-quoteHeader">
+                <img class="PullQuote-image" src="/static/imgs/quote.svg" alt="Quotation mark.">
+            </div>
+            <p class="PullQuote-body">
+                我的任务是构建我们的 CLI 工具，并发现了两个非常棒的项目，Cobra 和 Viper ，它们让开发 CLI 更容易。就个人而言，他们非常强大，非常灵活，非常擅长做这些事情。并且他们都能帮助你展示你的下一个 CLI 应用程序。
+            </p>
+            <div class="PullQuote-author">
+                <a href="https://medium.com/@skdomino/writing-better-clis-one-snake-at-a-time-d22e50e60056" target="_blank">&mdash; Steve Domino, 斯特拉拉高级工程师和设计师</a></div>
+        </div>
+
+        <p>
+            <a href="https://pkg.go.dev/github.com/spf13/cobra?tab=overview">Cobra</a> 既是一个创建强大的现代的 CLI 应用程序的库，也是一个使用 Go 生成普通应用程序和 CLI 应用程序的程序。Cobra 支持大多数流行的 Go 应用程序，包括 CoreOS， Delve， Docker， Dropbox， Git Lfs， Hugo， Kubernetes， 以及 <a href="https://pkg.go.dev/github.com/spf13/cobra?tab=importedby" rel="noreferrer" target="_blank">其他更多</a>. 使用集成的命令帮助，自动完成和文档 “这使得记录每个命令都变得非常简单，” <a href="https://blog.alexellis.io/5-keys-to-a-killer-go-cli/" rel="noreferrer" target="_blank">Alex Ellis</a> 说到， OpenFaaS 的创始人。
```

Comment:
```suggestion
            <a href="https://pkg.go.dev/github.com/spf13/cobra?tab=overview">Cobra</a> 既是一个创建强大的现代的 CLI 应用程序的库，也是一个使用 Go 生成普通应用程序和 CLI 应用程序的程序。Cobra 支持大多数流行的 Go 应用程序，包括 CoreOS、Delve、Docker、Dropbox、Git LFS、Hugo、Kubernetes 以及 <a href="https://pkg.go.dev/github.com/spf13/cobra?tab=importedby" rel="noreferrer" target="_blank">其他更多</a>。使用集成的命令帮助，自动完成和文档 “这使得记录每个命令都变得非常简单，” <a href="https://blog.alexellis.io/5-keys-to-a-killer-go-cli/" rel="noreferrer" target="_blank">Alex Ellis</a> 说到，OpenFaaS 的创始人。
```

“使用集成的命令帮助，自动完成和文档”这个好像有点问题，原文是什么？

Reply:
原文：
With integrated command help, autocomplete and documentation “[it] makes documenting each command really simple,” says Alex Ellis, founder of OpenFaaS.

修改成: 
有了集成的命令帮助、自动完成和文档化，"使每个命令的文档化变的非常简单"

Reply:
好多了！